src/corefx/System.Globalization.Native/CMakeLists.txt

```diff
@@ -6,12 +6,25 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 add_definitions(-DPIC=1)
 add_definitions(-DBIT64=1)
 
-find_library(ICUUC NAMES icuuc)
+find_library(ICUUC NAMES icuuc PATHS "/usr/local/opt/icu4c/lib")
 if(ICUUC STREQUAL ICUUC-NOTFOUND)
-    message(WARNING "Cannot find libicuuc, skipping build for System.Globalization.Native. .NET globalization is not expected to function. Try installing libicu-dev (or the appropriate package for your platform)")
+    message(ERROR "Cannot find libicuuc, skipping build for System.Globalization.Native. .NET globalization is not expected to function. Try installing libicu-dev (or the appropriate package for your platform)")
     return()
 endif()
 
+find_library(ICUI18N NAMES icui18n PATHS "/usr/local/opt/icu4c/lib")
+if(ICUI18N STREQUAL ICUI18N-NOTFOUND)
+    message(ERROR "Cannot find libicui18n, skipping build for System.Globalization.Native. .NET globalization is not expected to function. Try installing libicu-dev (or the appropriate package for your platform)")
+    return()
+endif()
+
+find_path(UTYPES_H "unicode/utypes.h" PATHS "/usr/local/opt/icu4c/include")
+
```

Comment:
(nitpick)  The other two places don't have a blank line between the "find_" command and the "if"